src/components/TextInput/TextInput.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../styles/variables.scss';
 /* @TODO -- replace any remaining harcoded values with variables or use global utility classes */
-.input {
+.TextInput {
```

Comment:
CSS Modules might not be necessary. We just need to scope our styles well. Perhaps, for each component, we prefix or suffix the class name with `palmettoComponents` to make library components easier to identify if someone is picking through the DOM in the browser? I don't know, just a half thought.